### PR TITLE
Feature flag for system management

### PR DESCRIPTION
**Pull request by wittejm**
First bit of design indecision: do we navigate to the two views via a single button in the menu? or two? If it's one button, what will navigation between the two views look like? Favorite solution: a submenu!
Screen Shot 2022-08-18 at 3 48 47 PM

### CODE REVIEW

File: .github/workflows/build-test-deploy.yml [hunk 1]
```diff
@@ -52,6 +53,7 @@ jobs:
           REACT_APP_API_URL: "https://connect-api.ameelio.org/api/v1/"
           REACT_APP_SENTRY_DSN: "https://af36ac2171b4413da95f553d5bdfc5df@o434922.ingest.sentry.io/6021906"
           REACT_APP_ENABLE_PROVIDERS: ""
+          REACT_APP_ENABLE_SYSTEM_MANAGEMENT: ""
```
Comment (cainlevy) [inline .github/workflows/build-test-deploy.yml:56]: this has a few more steps than you'd guess: https://github.com/Ameelio/connect-doc-client#adding-an-environment-variable

File: src/components/Menu/Menu.tsx [hunk 2]
```diff
@@ -172,6 +172,11 @@ export default function Menu({
           </AntdMenu.Item>
         ))}
       </SubMenu>
+      {featureEnabled("SYSTEM_MANAGEMENT") && (
+        <AntdMenu.Item onClick={() => window.alert("system management!")}>
+          <span>System Management</span>
+        </AntdMenu.Item>
```
Comment (cainlevy) [inline src/components/Menu/Menu.tsx:178]: should this include the submenu?

i do think submenus are most consistent with the doc client's navigation setup.

**Review by cainlevy — CHANGES_REQUESTED** [review]

File: src/components/Menu/Menu.tsx (hunk 2 above)
Comment (wittejm) [reply src/components/Menu/Menu.tsx]: yep I forgot to push my last commit :/

**Comment by jackofsometrade** [comment]
Sub menu looks good to me!  👍

**Review by cainlevy — APPROVED** [review]

**Review by jackofsometrade — APPROVED** [review]

**Pull request merged**